modules/resources/templates/training-cti.html

```diff
@@ -9,276 +10,130 @@
+    <li class="breadcrumb-item"><a href="/resources">Resources</a></li>
+    <li class="breadcrumb-item">
+      <a href="/resources/training">ATT&CK Training</a>
+    </li>
+    <li class="breadcrumb-item">CTI Training</li>
+  </ol>
+</div>
 
-                                    <ul>
-                                        <li><a href="/docs/training-cti/Making%20Defensive%20Recommendations%20Unguided%20Exercise.rtf" target="_blank">Making Defensive Recommendations Unguided Exercise</a>
-                                        </li>
-                                            <em>Provides steps for making defensive recommendations from ATT&CK techniques.
-                                            </em>
-                                    </ul>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+<div class="container pb-3">
+  <div class="getting-started">
+    <div class="clearfix pb-3">
+      <div class="clearfix">
+        <div class="card card-filter jump-to-section">
+          <div class="card-body">
+            Jump to Section
+            <ul>
+              {% for training in parsed.cti_training.values() %}
+              <li>
+                <a href="#{{training.id}}">{{training.short_name}}</a>
+              </li>
+              {% endfor %}
+            </ul>
+          </div>
+        </div>
+        <div class="getting-started-content">
+          <h1>CTI Trainings</h1>
+          <p>
+            The goal of this training is for students to understand the following:
+          </p>
+          <ul>
+            <li>
+              What ATT&CK is and why it’s useful for cyber threat intelligence
+              (CTI)
+            </li>
+            <li>
+              How to map to ATT&CK from both finished reporting and raw data
+            </li>
+            <li>
+              Why it’s challenging to store ATT&CK-mapped data and what you should
+              consider when doing that
+            </li>
+            <li>How to perform CTI analysis using ATT&CK-mapped data</li>
+            <li>How to make defensive recommendations based on CTI analysis</li>
+          </ul>
+          <p>
+            The modules below come with exercises intended to increase and test
+            your understanding of ATT&CK techniques and applications. It is
+            specified when are unguided (not walked through in the video or
+            reports).
+          </p>
+          <h2>Modules</h2>
+          {% for training in parsed.cti_training.values() %}
+          <div class="anchor" id="{{training.id}}"></div>
+          <div class="card w-60 card-training">
+            <div class="card-body training-card-body">
+              <div class="training-header">
+                <div class="training-text-container">
+                  <strong class="title">{{ training.name }}</strong>
+                  <div class="date">{{ training.module }}</div>
```

Comment:
I would recommend using the `text-label` class here rather than the `date` class. The `text-label` class transforms text to uppercase and will keep the module subheader more consistent with other pages.